app/components/delivery_partners/participants/table_row.rb

```diff
@@ -5,7 +5,11 @@ module Participants
     class TableRow < BaseComponent
       with_collection_parameter :participant_profile
 
-      delegate :user, :teacher_profile, :school_cohort, :schedule, :training_status, :ecf_participant_eligibility,
+      delegate :user,
+               :teacher_profile,
+               :training_status,
```

Comment:
you want to be using `training_status` on the `InductionRecord`. The profile one will start to no longer accurately reflect the user circumstances

Reply:
so if you are a DP you should be able to work out the lead provider and therefore work out the correct induction record